internal/common/filespec.go

```diff
@@ -219,7 +226,25 @@ func (this *FileSpec) CopyTarget(baseDir string, destDir string, onlyContents bo
 		// no-op.
 		return nil
 	case FILESPEC_TYPE_PATH:
-		return this.copyPath(baseDir, destDir, onlyContents)
+		if !hasGlobPattern(this.Path) {
+			return this.copyPath(baseDir, destDir, onlyContents)
+		}
+
+		files, err := this.matchFiles()
+		if err != nil {
+			return fmt.Errorf("Failed to resolve glob in path '%s': '%w'.", this.Path, err)
+		}
+
+		// Loop over each matched file and create a temporary FileSpec to copy it to the destination.
+		for _, file := range files {
+			tempFileSpec := &FileSpec{Type: FILESPEC_TYPE_PATH, Path: file}
```

Comment:
Why have a temp FileSpec?

Just so you can get access to copyPath()?
If so, then what you are doing here should "smell" to you.
You are wrapping an object to call a method that is in the package/file/type that you are currently working on.
You are making a work around for your own code.
Just change it so you have access to the method you need.